stdlib/sys.mli

```diff
@@ -411,7 +411,9 @@ external opaque_identity : 'a -> 'a = "%opaque"
 (** For the purposes of optimization, [opaque_identity] behaves like an
     unknown (and thus possibly side-effecting) function.
 
-    At runtime, [opaque_identity] disappears altogether.
+    [opaque_identity] creates zero runtime overhead, but does prevent the
+    argument from being garbage collected until after the location the call
+    appears.
```

Comment:
Small nitpick: isn't preventing the argument from being garbage collected a runtime overhead?

I'd keep "At runtime, ..." and make it distinct sentences

Reply:
Agreed.  I have made this change.